librz/bin/p/bin_avr.c

```diff
@@ -197,7 +197,7 @@ static RzList /*<RzBinSymbol *>*/ *symbols(RzBinFile *bf) {
 
 static RzList /*<RzBinString *>*/ *strings(RzBinFile *bf) {
 	// we dont want to find strings in avr bins because there are lot of false positives
```

Comment:
shall this be removed? are there still a lot of false positives?

Reply:
actually it does find the strings correctly for our tests.

Reply:
I will keep this  as is, since it does not break the tests (without it some stuff gets auto-detected and therefore some tests "fails").